bin/simulate.js

```diff
@@ -28,7 +28,9 @@ const largeEmailEvent = {
   from: 'campaigns@rabblerouser.team',
   to: ['pam@example.com'],
   subject: 'This email body is coming to you from S3!',
-  bodyLocation: 'email-from-john',
+  bodyLocation: {
+    key: 'email-from-john',
+  },
```

Comment:
This will require a change to the `group-mailer` right? It will need to publish the `send-email` event with this new structure?

Reply:
I've made the matching change to the `group-mailer`.